src/SimpleThings/EntityAudit/Resources/views/Audit/compare.html.twig

```diff
@@ -1,5 +1,21 @@
 {% extends "SimpleThingsEntityAuditBundle::layout.html.twig" %}
 
+{% macro showValue(value) %}
+    {% if value.timestamp is defined %}
+        {{ value|date('m/d/Y') }}
+    {% elseif value is iterable %}
+        <ul>
+        {% for element in value %}
+            <li>{{ element }}</li>
+        {% endfor %}
+        </ul>
+    {% else %}
+        {{ value }}
+    {% endif %}
+{% endmacro %}
+
+{% import _self as helper %}
```

Comment:
Ah I see, but that's for the `1.x` branch. Correct me if I'm wrong, but it still won't work in the `2.x` branch as mentioned before.